docs/docs/how-to-guides/dynamicContent.md

```diff
@@ -0,0 +1,93 @@
+# Rendering Dynamic Content
+
+Static site generation is awesome because of the performace and stability it offers. However, an ecommerce is much more than a static product catalog. Ecommerces offer the possibility of creating a personalized experience for the final customer by offering personalized prices, availability and search results.
```

Comment:
```suggestion
Static Site Generation (SSG) is awesome because of the performance and stability it offers. However, an ecommerce is much more than a static product catalog. Ecommerces offer the possibility of creating a personalized experience for the final customer by offering personalized prices, availability and search results.
```